src/TwigComponent/doc/index.rst

```diff
@@ -670,6 +697,10 @@ You can also add more, named blocks:
         </twig:block>
     </twig:Alert>
 
+.. note::
+
+    Because your TwigComponent live inside his own specific context, you can't use the parent function inside the block of your component template.
+
```

Comment:
This is not true :) 

https://symfony.com/bundles/ux-twig-component/current/index.html#passing-html-to-components-via-blocks

Reply:
Maybe my formulation is bad but I was trying to explain the issue @Nek- had. How to you explain that in a few lines?

Reply:
You should maybe say that parent refers to its own context instead.